packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -29,81 +33,81 @@ module.exports = function (defaults) {
   });
 }
 
-async function classicInRepoAddon(project) {
-  await classic(project);
-  await addInRepoAddon(project, 'fake-addon');
-}
-
-async function embroider(project) {
-  project.linkDependency('@embroider/core', {
-    baseDir: __dirname,
-  });
-  project.linkDependency('@embroider/compat', {
-    baseDir: __dirname,
-  });
-  project.linkDependency('@embroider/webpack', {
-    baseDir: __dirname,
-  });
-  project.linkDependency('webpack', {
-    baseDir: __dirname,
-  });
-
-  merge(project.files, {
-    'ember-cli-build.js': `'use strict';
-
-const EmberApp = require('ember-cli/lib/broccoli/ember-app');
-
-module.exports = function (defaults) {
-  let app = new EmberApp(defaults, {
-    babel: {
-      plugins: [
-        [
-          require.resolve('babel-plugin-ember-test-metadata'),
-          { enabled: true }
-        ]
-      ],
-    }
-  });
-
-  const { Webpack } = require('@embroider/webpack');
-  return require('@embroider/compat').compatBuild(app, Webpack);
-};
-`,
-  });
-}
-
-async function embroiderInRepoAddon(project) {
-  await embroider(project);
-  await addInRepoAddon(project, 'fake-addon');
-}
-
-async function addInRepoAddon(project, name, version = '0.0.0') {
-  project.linkDependency('ember-add-in-repo-tests', {
-    baseDir: __dirname,
-  });
-
-  project.pkg['ember-addon'] = {
-    paths: [`lib/${name}`],
-  };
-
-  merge(project.files, {
-    lib: {
-      [name]: {
-        'package.json': {
-          name,
-          version,
-          keywords: ['ember-addon'],
-        },
-        'index.js': `module.exports = {
-          name: require("./package").name,
-        };`,
-        tests: {
-          unit: getTestFiles('with-hooks-test.js'),
-        },
-      },
-    },
-  });
-}
+// async function classicInRepoAddon(project) {
```

Comment:
Just a preference thing, but I think it's fine to keep these uncommented and just let them run red 🤷‍♂️